tests/mock_controller_test.py

```diff
@@ -11,7 +11,7 @@ class MockTestCase(asynctest.TestCase):
         self.port = 5000
         self.mock_ctrl = None
 
-        self.mock_ctrl = dcs.MockDomeController(port=self.port)
+        self.mock_ctrl = Dome.MockDomeController(port=self.port)
         asyncio.create_task(self.mock_ctrl.start())
         await asyncio.sleep(1)
```

Comment:
I will also mention here that a pattern I have found useful is to *always* have a task for any background jobs your code runs. That way at shutdown you can simply cancel all these tasks to shut down the background jobs. (For TCP/IP communication you may want to be a bit more careful so as to not leave a partial read or write; I tend to have a task that waits for the next reply and cancel that instead cancelling the task for the "read command" loop, but that may unnecessary).

I usually initialize these tasks to salobj.make_done_future() for asyncio.Future() (depending if they should be initialized as done or not done) so that I can cancel them without checking if they are None (or done -- cancel always works).

See https://ts-salobj.lsst.io/salobj_cscs.html#telemetry-loop-example